source/client/output_formatter_impl.cc

```diff
@@ -393,5 +394,24 @@ const nighthawk::client::DurationHistogram FortioOutputFormatterImpl::renderFort
   return fortio_histogram;
 }
 
+std::string
+FortioPedanticOutputFormatterImpl::formatProto(const nighthawk::client::Output& output) const {
+  std::string res = FortioOutputFormatterImpl::formatProto(output);
+  // clang-format off
+  // Fix two types of quirks. We disable linting because we use std::regex directly.
+  // This should be OK as the regular expression we use can be trusted.
+  // 1. We misdefined RequestedRPS as an int, whereas Fortio outputs that as a string.
+  res = std::regex_replace(res, std::regex(R"EOF("RequestedQPS"\: ([0-9]*))EOF"),
```

Comment:
(optional) Would you mind including (in the comment) and example of the matched string? That usually helps when trying to understand regexes in code. This one is fairly trivial, so optional, but we could use one for the second regex.

Reply:
Yeah, I always felt that regular expressions are kind of a write-only concept :-)
Done in 2df8c2be